webview.h

```diff
@@ -1353,52 +1353,44 @@ class webview : public browser_engine {
   }
 
   using binding_t = std::function<void(std::string, std::string, void *)>;
-  using binding_ctx_t = std::pair<binding_t *, void *>;
+  class binding_ctx_t {
+  public:
+    binding_ctx_t(binding_t *f, void *a, bool s = true)
+        : callback(f), arg(a), sync(s) {}
+    binding_t *callback;
+    void *arg;
+    bool sync;
+  };
```

Comment:
Could you please add brief comments for the member variables in `binding_ctx_t` to make it easier to understand their purpose?